NEWS.md

```diff
@@ -17,6 +17,855 @@
 
 MXNet Change Log
 ================
+## 1.5.0
```

Comment:
This TOC should work... but we might need to remove the old TOC for 1.4.1 (or update it) so there are no conflicts.
```suggestion
- [New Features](#new-features)
  * [Automatic Mixed Precision(experimental)](#automatic-mixed-precision-experimental-)
  * [MKL-DNN Reduced precision inference and RNN API support](#mkl-dnn-reduced-precision-inference-and-rnn-api-support)
  * [Dynamic Shape(experimental)](#dynamic-shape-experimental-)
  * [Large Tensor Support](#large-tensor-support)
  * [Dependency Update](#dependency-update)
  * [Gluon Fit API(experimental)](#gluon-fit-api-experimental-)
  * [New Operators](#new-operators)
- [Feature Improvements](#feature-improvements)
  * [Operators](#operators)
  * [MKLDNN](#mkldnn)
  * [ONNX](#onnx)
  * [TensorRT](#tensorrt)
  * [FP16 Support](#fp16-support)
  * [Deep Graph Library(DGL) support](#deep-graph-library-dgl--support)
  * [Horovod Integration](#horovod-integration)
  * [Dynamic Shape](#dynamic-shape)
  * [Backend Engine](#backend-engine)
  * [Large Tensor Support](#large-tensor-support-1)
  * [Quantization](#quantization)
  * [Profiler](#profiler)
  * [CoreML](#coreml)
- [Front End API](#front-end-api)
  * [Gluon](#gluon)
  * [Python](#python)
- [Language Bindings](#language-bindings)
  * [Scala](#scala)
  * [Java](#java)
  * [C++](#c--)
  * [Clojure](#clojure)
  * [Julia](#julia)
  * [Perl:](#perl-)
  * [R](#r)
- [Performance Improvements](#performance-improvements)
- [Example and Tutorials](#example-and-tutorials)
- [Website](#website)
- [Documentation](#documentation)
- [Build and Test](#build-and-test)
- [Bug-fixes](#bug-fixes)
- [License](#license)
- [Depreciations](#depreciations)
- [Known Issues](#known-issues)

## 1.5.0
```